src/Compilers/Core/Portable/Operations/IPatternOperation.cs

```diff
@@ -15,6 +15,10 @@ namespace Microsoft.CodeAnalysis.Operations
     /// </remarks>
     public interface IPatternOperation : IOperation
     {
+        /// <summary>
+        /// The input type to the pattern-matching operation.
+        /// </summary>
+        ITypeSymbol InputType { get; }
```

Comment:
>InputType [](start = 20, length = 9)

It doesn't look like any asserts were added about this property in TestOperationVisitor.